pkgs/misc/base16-shell-preview/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "base16-shell-preview";
+  version = "0.3.0";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "base16_shell_preview";
+    sha256 = "0x2fbicrcqgf2dl7dqzm14dz08vjjziabaaw33wah3v9wv4rw7jq";
+  };
+
+  # Added to prevent test from failing due to using "/homeless-shelter" as HOME
+  checkPhase = "HOME=$TMPDIR";
```

Comment:
I've looked in the tarball and repo, and I'm not seeing any tests.

Currently by doing this, the `checkPhase` just sets `HOME` and exits.

Reply:
You're probably getting this error when tests are enabled
because it's just trying to run the script
https://github.com/nvllsvm/base16-shell-preview/blob/master/base16_shell_preview/__init__.py#L8

Reply:
yep, and it seems I'm able to install it successfully with explicitly setting the HOME.  But in any case, since there's no tests, its better to just revert to disabling the check?

Reply:
Yep, no tests no `checkPhase` needed.